content/rs/administering/designing-production/networking/port-configurations.md

```diff
@@ -29,6 +29,8 @@ update your firewall with the port for that new database endpoint.
 | 8001 | Used by your application to access the RS [Discovery Service]({{< relref "/rs/concepts/data-access/discovery-service.md" >}}) | TCP |
 | 8443 | Used for secure (https) access to the management web UI | TCP |
```

Comment:
what does each row represent? 
I think each row could represent a system.
if this case:

- 8443 and 8444 should be in a row (CM)
- 9080 should be with cnm_api row (8080, 9443) and it's name should be cluster API

Reply:
@RoeyPrat Agreed. Can you make that commit?

Reply:
sure, but in a ticket - created RED-26867.
I suggest merging this for now, as it is an improvement on the existing.